btr-web/btr-main-app/utils/btr-bods/si-to-btr-bods-converters.ts

```diff
@@ -105,65 +106,94 @@ const _createInterestSharesVotes = (
     endDate: si.endDate
   }
 }
-const _addVotes = (interest: BodsInterestI, maximum: number, sharesOrVotes: BodsInterestTypeE) => {
+
+// const _addVotes = (interest: BodsInterestI, maximum: number, sharesOrVotes: BodsInterestTypeE) => {
+//   interest.share = {
+//     maximum,
+//     exclusiveMaximum: false
+//   }
+//   interest.type = sharesOrVotes
+// }
+
+const _getPercentageRange = (interest: BodsInterestI, range: PercentageRangeE, sharesAndVotes: BodsInterestTypeE) => {
   interest.share = {
-    maximum,
+    exclusiveMinimum: true,
     exclusiveMaximum: false
   }
-  interest.type = sharesOrVotes
+
+  interest.type = sharesAndVotes
+
+  switch (range) {
+    case PercentageRangeE.LESS_THAN_25:
```

Comment:
exclusiveMinimum should also be set to 'false' here? And exclusiveMaximum should be set to true?